python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py

```diff
@@ -510,3 +514,334 @@ def _get_log_path(self, params: PipesParams) -> Optional[str]:
                 self.log_path = f"dbfs:{match.path}"
 
         return self.log_path
+
+
+# ####################################
+# ##### Databricks Serverless
+# ####################################
+
+
+class PipesDatabricksServerlessClient(PipesClient, TreatAsResourceParam):
+    """Pipes client for Databricks Serverless.
+
+    Args:
+        client (WorkspaceClient): A databricks `WorkspaceClient` object.
+        context_injector (Optional[PipesContextInjector]): A context injector to use to inject
+            context into the k8s container process. Defaults to :py:class:`PipesUnityCatalogVolumesContextInjector`.
+        message_reader (Optional[PipesMessageReader]): A message reader to use to read messages
+            from the Databricks Serverless job. Defaults to :py:class:`PipesUnityCatalogVolumesMessageReader`.
+        poll_interval_seconds (float): How long to sleep between checking the status of the job run.
+            Defaults to 5.
+        forward_termination (bool): Whether to cancel the Databricks Serverless job if the orchestration process
+            is interrupted or canceled. Defaults to True.
```

Comment:
Need docstring for `volume_path`

Reply:
https://github.com/dagster-io/dagster/pull/31451